html/semantics/text-level-semantics/the-a-element/a-download-click-404.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<title>Clicking on an &lt;a> element with a download attribute and href that leads to 404 should not navigate</title>
+<link rel="author" title="phistuck" href="mailto:user@domain.com">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/#the-a-element:activation-behaviour">
+<link rel="help" href="https://html.spec.whatwg.org/multipage/links.html#attr-hyperlink-download">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<iframe id="error-frame" src="resources/a-download-404.html"></iframe>
+<script>
+"use strict";
+async_test(function (t) {
+    const errorFrame = document.querySelector("#error-frame");
+    errorFrame.addEventListener("load", function () {
```

Comment:
Thank you for the review, I forgot to write before. :)

Are you fine with timeout part, though?
Or since this is expected to run mostly locally, then it is not a problem in this case?
Should I make it shorter?